HelpSource/Classes/EventPatternProxy.schelp

```diff
@@ -113,13 +113,13 @@ subsection::a) embedding EventPatternProxy in streams
 
 code::
 (
-SynthDef("Pdefhelp", { arg out, freq, sustain=1, amp=1, pan;
+SynthDef("Pdefhelp") {
```

Comment:
This is correct syntax but a little surprising perhaps for beginners? Also the amp should be 0.1 by cnvention, this was also wrong in the current version. I think the best would be:
```
(
SynthDef(\Pdefhelp, { |out, freq, sustain=1, amp=0.1, pan|
	var env, u=1;
	env = EnvGen.kr(Env.perc(0.03, sustain), doneAction: Done.freeSelf);
	5.do { var d; d = exprand(0.01, 1); u = SinOsc.ar(d * 300, u, rrand(0.1,1.2) * d, 1) };
	Out.ar(out, Pan2.ar(SinOsc.ar(u + 1 * freq, 0, amp * env), pan));
}).add
)

Reply:
Thanks!!!!

Sure, I'm sorry, that's some personal love issues with the function-arguments-can-be-outside-parens syntax. I like it so much that I have hard time to see the world otherwise :) I'll put those functions back inside those parens, and change amps to 0.1 since we are here!

Reply:
I totally understand. I am also somewhat attracted to `ar(SinOsc, 440)`, and I suspect you are tempted to do `ar(SinOsc) { 440 }`